views/review/today.html.twig

```diff
@@ -30,132 +40,138 @@
         <li role="presentation"><a href="{{ path('review_measurementsRecentModify') }}">Recently Modified Physical Measurements</a></li>
     </ul>
```

Comment:
Can we make the Today's Participant tab clickable (i.e. add the link to the href) so that the user can click on the tab to reset to today's view?